.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Use Markdown language for styling.
+Github's Markdown Guide https://guides.github.com/features/mastering-markdown/
+Please use the SEARCH function before opening a new issue!
+-->
+
+### --Issue Summary--
+<!-- A brief overview of your issue  -->
+
+### --Video or Image Reference--
+<!-- (Optional) A screenshot, short video, etc. to show your problem -->
+
+### --Steps to reproduce--
+<!-- Describe the steps to reproduce the problem, more steps are better. -->
+
+
+### --System Information--
++ OpenToonz Version:
+<!-- You can find the VERSION in Menu->Help->About OpenToonz -->
+
++ Operating System:
+<!-- (e.g., Windows10, macOS 10.12, Ubuntu 16.04, Debian Stretch) -->
+
++ RAM Size:
+<!-- (4GB, 8GB etc.) -->
+
++ Graphics Tablet:
+<!-- (Optional) (Write your MODEL, e.g., WACOM Intuos 2) -->
+
+### --Actual Results--
+<!-- Explain what you actually got. -->
+
+### --Expected Results--
+<!-- Explain how the program should behave in the context you're using it. -->
```

Comment:
How about moving `Actual Results` and `Expected Results` to just under `Step to reproduce` , since they are related?

Reply:
I had those last as sometimes they become quite long and the other info can provide a quick summary without having to look over long cut-n-pasted fields. It's a minor consideration and is easily changed back if cut-n-pastes happen often.